setup.py

```diff
@@ -18,11 +18,13 @@
     long_description_content_type="text/markdown",
     entry_points={'console_scripts': [
         'geometric-optimize = geometric.optimize:main',
+        'geometric-neb = geometric.neb:main'
     ]},
     install_requires=[
         'numpy>=1.11',
         'networkx',
         'six',
+        'scipy',
```

Comment:
I am unsure if the scipy dependency is needed because it is only used to call `sqrtm()` in a function whose use is unclear.  Please check to see whether this can be removed.

Reply:
I deleted it.